packages/actor-init-query/lib/cli/CliArgsHandlerQuery.ts

```diff
@@ -80,6 +80,11 @@ export class CliArgsHandlerQuery implements ICliArgsHandler {
             'physical',
           ],
         },
+        localizeBlankNodes: {
+          type: 'boolean',
+          describe: `Set the blank node localization.
```

Comment:
Suggestion for a shorter one-liner: "If blank nodes should be localized per bindings entry"